rcl/test/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ find_package(osrf_testing_tools_cpp REQUIRED)
 find_package(rcpputils REQUIRED)
 find_package(rcutils REQUIRED)
 find_package(rmw_implementation_cmake REQUIRED)
+find_package(rosidl_runtime_cpp REQUIRED)
```

Comment:
For future readers; I was slightly concerned that this was going to introduce a new (test) dependency on this package, but it turns out that `rcl` already indirectly depends on `rosidl_runtime_cpp`, so no worries there.